engines/osxnative/sandbox.go

```diff
@@ -52,7 +52,7 @@ type stderrLogWriter struct {
 }
 
 func (w stderrLogWriter) Write(p []byte) (int, error) {
-	w.context.LogError(string(p))
+	w.context.LogRaw(string(p))
```

Comment:
hmm... Maybe I should kick myself for not seeing this earlier... but you want a `io.Writer` interface.. you don't need to wrap `TaskContext` the `LogError` and `Log` methods are really only for non-raw messages...

The raw messages should go to `TaskContext.LogDrain()` which returns an `io.Writer` that is buffered and all..